cli/formatter/container.go

```diff
@@ -0,0 +1,56 @@
+package formatter
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
+)
+
+//ContainerHeader is the map to show container head
+type ContainerHeader map[string]string
+
+//ContainerContext is the map to show container context detail
+type ContainerContext map[string]string
+
+//NewContainerHeader is to generate a ContainerHeader to be show
+func NewContainerHeader() ContainerHeader {
+	//need to specify how the Mounts shows
+	//if need to show another attr of container.add vaule in the map
+	containerHeader := ContainerHeader{
+		"Name":    "Name",
+		"ID":      "ID",
+		"Status":  "Status",
+		"Created": "Created",
+		"Image":   "Image",
+		"Runtime": "Runtime",
+		"Command": "Command",
+		"ImageID": "ImageID",
+		"Labels":  "Labels",
+		//"Mounts":  "Mounts",
+		"State": "State",
+	}
+	return containerHeader
+}
+
+//NewContainerContext is to generate a ContainerContext to be show
+func NewContainerContext(c *types.Container, flagNoTrunc bool) ContainerContext {
```

Comment:
the container context's key is same with the container head?

Reply:
yes, they have the same keys.